sway/commands/move.c

```diff
@@ -918,11 +918,17 @@ struct cmd_results *cmd_move(int argc, char **argv) {
 
 	if (strcasecmp(argv[0], "window") == 0 ||
```

Comment:
Technically we need to make sure `argc > 0` before calling `strcasecmp` here, to prevent commands ending with `--no-auto-back-and-forth` from segfaulting.